operations/mimir/query-frontend.libsonnet

```diff
@@ -21,7 +21,7 @@
       'query-frontend.max-cache-freshness': '10m',
 
       // Limit queries to 500 days, allow this to be override per-user.
```

Comment:
```suggestion
      // Limit queries to 500 days; allow this to be overridden per user.
```

Reply:
Rationale: matches phrasing of https://github.com/grafana/mimir/pull/3879/files#diff-f600bfee9a33d23b0945bda697930b5d1805149ab3adda8f63458073b3988a1bR280.

Reply:
Just to be 100% clear, "user" means a human here, right?